src/ripple/nodestore/impl/codec.h

```diff
@@ -301,13 +301,7 @@ template <class = void>
 void const*
 zero32()
 {
-    static std::array<char, 32> v =
-        []
-        {
-            std::array<char, 32> v;
-            v.fill(0);
-            return v;
-        }();
+    static std::array<char, 32> v{};
```

Comment:
Wow, good find!

Reply:
There's another one of these at src/ripple/protocol/impl/STTx.cpp:449.

Reply:
Thanks!